.github/CHANGELOG.md

```diff
@@ -7,6 +7,9 @@
 
 <h3>Improvements</h3>
 
+* Add QubitMatrix operator for custom operator implementation
+[(#1435)](https://github.com/PennyLaneAI/pennylane/pull/1435)
+
```

Comment:
Would be good to include a short example.

Reply:
💯

Reply:
example added here: c1a9cb2969654729bda9f53679f9978334184371